.github/pull_request_template.md

```diff
@@ -13,6 +13,15 @@ Addresses #*PR# HERE*
 
 ## Changelog
 
+<!--
+  Do not change this section's content without updating
+  - .github/workflows/changelog.yml
```

Comment:
Hmm this note will appear also when somebody opens a PR, I wonder if this could be confusing as they could think this note is for the PR. Perhaps updating to something like "Do not change this section's content in the github template..."? Or add aditional explanations that this warning is for the ones updating the gh template and not for the ones opening a PR

Reply:
Ah yes, that'd be confusing, I really didn't realize that when updating the template. Let me fix that. Will push here.

Reply:
I simply reverted it - it feels confusing even with explanation - probably better to say less here. I will rely that we will know about this action and there's also `<!-- [DO NOT REMOVE-USED BY GH ACTION] CHANGELOG START -->` that gives a hint :)

Reply:
Thanks @AlexVelezLl